### ISSUE TITLE
Why doesn't I001 allow multiple imports from the same package on one line?

### ISSUE THREAD

**Issue by llamafilm** [astral-sh/ruff#21507]
### Question

Ruff rule I001 is flagging when I have multiple imports from the same package on one line.  Why does it do this?  If the goal is to follow PEP 8, it gives this example:

```py
# Correct:
import os
import sys

# Wrong:
import sys, os

# Correct:
from subprocess import Popen, PIPE
```

Ruff says that last line is wrong.

### Version

ruff 0.14.4

**Comment by amyreese** [comment]
Ruff uses case-sensitive sorting for imported names, so it wants to reorder those names:

```
$ uvx ruff check --select I001 --diff foo.py
--- foo.py
+++ foo.py
@@ -1 +1 @@
-from subprocess import Popen, PIPE
+from subprocess import PIPE, Popen

Would fix 1 error.
> [1]
```

**Comment by llamafilm** [comment]
That doesn't work for me.

```
% uvx ruff check --select I001 --diff 21507.py
--- 21507.py
+++ 21507.py
@@ -1,2 +1,3 @@
-from subprocess import PIPE, Popen
+from subprocess import PIPE
+from subprocess import Popen
 

Would fix 1 error.
```

**Comment by MichaReiser** [comment]
Can you run `uvx ruff check --show-settings 21507.py` and share the output with us?

**Comment by llamafilm** [comment]
Ok I found it!  Thanks for pointing me in the right direction.  It was caused by this in pyproject.toml, a default choice from django-cookiecutter.

```toml
[tool.ruff.lint.isort]
force-single-line = true
```
